build.gradle.kts

```diff
@@ -15,6 +16,7 @@ allprojects {
     repositories {
         google()
         mavenCentral()
+        //maven("https://s01.oss.sonatype.org/content/repositories/comapollographql-1006")
```

Comment:
```suggestion
```

Reply:
I kept it on purpose for myself to remember where to add this line next time but you're right that doesn't belong to a tutorial, I'll remove it.